blambda/deploy.py

```diff
@@ -267,7 +267,7 @@ def publish(name, role, zipfile, options, dryrun):
          str: the arn of the new or updated function
     """
     client = clients.lambda_client
-
+    options.pop('name', None)
```

Comment:
I'm not a python expert, but don't see how this can change the name variable.  I see how either of the following could
`    name = options.pop('name', name)`
or 
`    name = options.get("name", name)`

Reply:
Discussed with @balihoo-vbreus , the goal is to just remove 'name' from options IF it exists.  Setting the name from options is done when setting manifest_func_name elsewhere